src/newbank/server/User.java

```diff
@@ -0,0 +1,42 @@
+package newbank.server;
+
+public abstract class User {
+
+	private String userID;
+	private String password;
+	
+	public User(String userID, String password) {
+		this.userID = userID.toLowerCase();
+		this.password = password;
+	}
+	
+	public String getUserType() {
+		return "generic";
```

Comment:
We can have this as abstract method. 
	`abstract public String getUserType() ;`
and not let a "generic" user type exist. The type will be implemented by the child classes

Reply:
Ok, I agree.